### ISSUE TITLE
[Crash]: Game crashes on startup, despite nothing changing from last launch to this one

### ISSUE THREAD

**Issue by dudeloony** [FTBTeam/FTB-University#223]
* What version of University: V1.1.1
* Have you added any mods: nope
* If yes, which mods:
* Crash report link (preferably Pastebin etc, not the file): https://gist.github.com/dudeloony/e52ebc3b184fa49627c1cef765d3644f 
* What were you doing when the crash happened: Starting up the game
* Any other details: The crash log makes me think it might be the walrus block from extra cells. I'm going to see if I can delete it somehow. And I did use to have optifine, but it caused so many problems that I got rid of it. That was several play sessions ago though, so I don't think it would be a cause.

**Comment by dudeloony** [comment]
I found a "Walrus Shrinker" mod. It has the Extra Cells 2 mod except with a smaller walrus file. At first it didn't work for me, but after a restart on my laptop, it worked like a charm and now I'm back to playing. I recommend adding the walrus shrinker mod for V 1.2. Here's the link: https://www.curseforge.com/minecraft/texture-packs/extra-cells-2-walrus-shrinker
P.S. It is also worth noting that I tried restarting without the walrus shrinker mod to no avail. So it appears that the walrus shrinker did have an effect in one way or another.
